webdriver/tests/perform_actions/wheel.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from webdriver.error import NoSuchWindowException
+
+from tests.perform_actions.support.refine import filter_dict, get_events
+from tests.support.asserts import assert_move_to_coordinates
+
+
+@pytest.fixture(autouse=True)
+def release_actions(session, request):
```

Comment:
The reason I overwrote it here is this fails right now, but I have fixed it here https://chromium-review.googlesource.com/c/chromium/src/+/2387546, which has not been released to the chromedriver version that is used to run this PR, but it should be released soon.